app/helpers/application_helper.rb

```diff
@@ -77,14 +77,9 @@ def icon_url(name)
   end
 
   def cloudinary(url, width = "500", quality = 80, format = "auto")
-    cl_image_path(url || asset_path("#{rand(1..40)}.png"),
-                  type: "fetch",
-                  width: width,
-                  crop: "limit",
-                  quality: quality,
-                  flags: "progressive",
-                  fetch_format: format,
-                  sign_url: true)
+    image_url = url || asset_path("#{rand(1..40)}.png")
```

Comment:
`url.presence` is probably more forward thinking as empty string would not result in the second clause to be called